packages/clerk-js/src/ui/elements/LineItems.tsx

```diff
@@ -48,17 +48,18 @@ interface GroupProps {
    */
   borderTop?: boolean;
   variant?: GroupVariant;
+  expand?: boolean;
 }
 
-function Group({ children, borderTop = false, variant = 'primary' }: GroupProps) {
+function Group({ children, borderTop = false, variant = 'primary', expand = false }: GroupProps) {
   return (
     <GroupContext.Provider value={{ variant }}>
       <Box
         elementDescriptor={descriptors.lineItemsGroup}
         elementId={descriptors.lineItemsGroup.setId(variant)}
         sx={t => ({
           display: 'grid',
-          gridTemplateColumns: 'repeat(2, minmax(0, 1fr))',
+          gridTemplateColumns: `repeat(${expand ? 1 : 2}, minmax(0, 1fr))`,
```

Comment:
Not super keen on the prop `expand`. Feels like title and description should have an sx prop that allows you to span multiple columns as needed. This makes it more flexible for responsive situations.